theme/documentation/article_pages.md

```diff
@@ -30,4 +30,6 @@ This article is to be used as the template for **Official Documentations**. Offi
 
 This template contains a navigation on the left side that allows the user access to more content. It displays all the section headings in the category of the current article.
 
-When a user clicks on a section relevant to their needs, a list of articles organized under that section appear. The current article is always expanded and highlighted.
\ No newline at end of file
+When a user clicks on a section relevant to their needs, a list of articles organized under that section appear. The current article is always expanded and highlighted.
+
+This template also contains a Table of Contents on the right hand panel that will highlight the current active heading as the user scrolls down the page.
```

Comment:
Can we change this language to:

`This template also contains a Table of Contents on the right hand panel highlighting the currently selected heading as the user scrolls down the page.`